pkg/cmd/scan.go

```diff
@@ -341,3 +353,13 @@ func parseOutputFlag(out string) (*output.OutputConfig, error) {
 		Options: options,
 	}, nil
 }
+
+func validateTfProviderVersionString(version string) error {
+	if version == "" {
+		return nil
+	}
+	if match, _ := regexp.MatchString("^\\d+\\.\\d+\\.\\d+$", version); !match {
+		return errors.Errorf("Invalid version argument %s, expected a valid semver string (e.g. 2.13.4)", version)
+	}
```

Comment:
Can't you use go-version to check the version here? https://github.com/cloudskiff/driftctl/blob/main/pkg/iac/terraform/state/versions.go#L6

Reply:
I was wondering if we can use this lib but I don't think validation match what we really want there https://github.com/hashicorp/go-version/blob/master/version_test.go#L49

The simple regex was more relevant IMHO for this usage.